<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="welcome-PC" name="TestSoftAssertions" tests="1" failures="1" timestamp="12 Jun 2018 06:19:44 GMT" time="0.148" errors="0">
  <testcase name="doRegisterTest" time="0.096" classname="testCases.TestSoftAssertions">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [xxxxxxx] but found [yyyyyyy],
	expected [c] but found [a]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [xxxxxxx] but found [yyyyyyy],
	expected [c] but found [a]
	at testCases.TestSoftAssertions.doRegisterTest(TestSoftAssertions.java:22)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- doRegisterTest -->
</testsuite> <!-- TestSoftAssertions -->
